src/server/game/Movement/MovementGenerators/PointMovementGenerator.h

```diff
@@ -64,9 +66,10 @@ class AssistanceMovementGenerator : public PointMovementGenerator<Creature>
 class EffectMovementGenerator : public MovementGenerator
 {
     public:
-        explicit EffectMovementGenerator(uint32 id) : _pointId(id) { }
+        explicit EffectMovementGenerator(uint32 id) : _pointId(id), _duration(0) { }
```

Comment:
Is this constructor used anywhere after this PR? If not then you can drop it (together with Optional use)

Reply:
I think I'll also move it to another file and rename